docs/content/modules/phase_field/systems/Materials/ElasticEnergyMaterial.md

```diff
@@ -1,9 +1,30 @@
-<!-- MOOSE Object Documentation Stub: Remove this when content is added. -->!devel /Materials/ElasticEnergyMaterial float=right width=auto margin=20px padding=20px background-color=#F8F8F8
-
+!devel /Materials/ElasticEnergyMaterial float=right width=auto margin=20px padding=20px background-color=#F8F8F8
 
 # ElasticEnergyMaterial
 !description /Materials/ElasticEnergyMaterial
 
+This material generates an elastic free energy contribution consistent with the TensorMechanics stress divergence kernels. This allows for proper coupling of the phase field problem and the mechanics problem. The contribution of the `ElasticEnergyMaterial` and the chemical free energy (possibly defined using a `DerivativeParsedMaterial`) are then summed up using a `DerivativeSumMaterial` to form a _total free energy_ which is passed to the phase field kernels.
```

Comment:
I bet we can do this automatically, look for "`ClassName`" and create the link if it finds a unique match.